bin/ruby-build

```diff
@@ -1040,6 +1043,18 @@ use_homebrew_yaml() {
   fi
 }
 
+use_freebsd_yaml() {
+  if [ "FreeBSD" = "$(uname -s)" ]; then
+    local release="$(uname -r)"
+    if [ "${release%%.*}" -ge 11 ]; then
+      if pkg info -e libyaml > /dev/null; then
+        package_option ruby configure --with-libyaml-dir="$(pkg info --prefix libyaml | cut -wf2)"
```

Comment:
Could we just get away with calling `pkg info --prefix libyaml` once instead of `pkg info -e libyaml && pkg info --prefix`?

And what is the full output of `pkg info --prefix` before it gets cut?